dclab/rtdc_dataset/config.py

```diff
@@ -75,7 +75,7 @@ def __setitem__(self, key, value):
                 typ = dfn.get_config_value_type(self.section, key)
                 if typ is not None and not isinstance(value, typ):
                     warnings.warn(
-                        f"Type of confguration key [{self.section}]: {key} "
+                        f"Type of configuration key [{self.section}]: {key} "
```

Comment:
By the way, I am getting the following warning after setting a `f1dfloatduple` for "focus interval":

`WrongConfigurationTypeWarning: Type of confguration key [qpi]: focus interval should be <class 'tuple'>, got <class 'numpy.ndarray'>!`

This happens here (presumably) when reading the file. This should not happen, right? If you write a tuple, and h5py reads this attr as a numpy array, that's not a wrong configuration type.

Reply:
Did you resolve this already with the new converter and test you implemented?